ruby/vampires.rb

```diff
@@ -0,0 +1,63 @@
+puts "How many employees are there ?"
+employees=gets.chomp.to_i
+for i in 1..employees do
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
+  garlic = gets.chomp
+  puts "Would you like to enroll in the company's health insurance?(y/n)"
+  insurance = gets.chomp
+
+  # p name, age, birth_year, garlic, insurance
+
+  probably_not_a_vampire=false;
+  #
+  # in hind sight probably should have broken age check
+  # logic out and added another boolean for age_check
+  #
+  if ((2017-birth_year-age).abs<=1) && ((garlic=="y") || (insurance=="y"))

```

Comment:
In order to DRY up your code, I'd remove the logic from the conditional statements. Boolean values are much easier to evaluate with conditional statements because there's only two options: true or false. You could remove the birth year/age logic and set a boolean variable to evaluate; same with garlic and insurance. Do the logic once to get a boolean and use that in your conditionals. For instance, the long form would be: 
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.